modal/_container_entrypoint.py

```diff
@@ -155,7 +155,7 @@ def run_input(input_id: str, function_call_id: str, args: Any, kwargs: Any) -> N
                 if inspect.iscoroutine(res) or inspect.isgenerator(res) or inspect.isasyncgen(res):
                     raise InvalidError(
                         f"Sync (non-generator) function return value of type {type(res)}."
-                        " You might need to use @stub.function(..., is_generator=True)."
+                        " You might need to use @app.function(..., is_generator=True)."
```

Comment:
Feels like we might want to reword this one to be more robust, e.g. "you might need to use `is_generator=True` in your function decorator"